src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -57,13 +208,29 @@ class Settings extends React.Component<SettingsProps, SettingsState> {
         const element = document.body;
         element.classList.toggle("darkMode");
         this.setState({ darkMode: darkMode });
-        window.localStorage.setItem("darkMode", String(darkMode));
     }
 
     render() {
         return (
-            <div className="w-full h-full">
-                <p className="text-white font-medium mt-6 ml-4 text-3xl">Inop.</p>
+            <div className="w-full h-full flex flex-col">
+                <div className="flex-grow m-6 rounded-xl flex flex-row">
+                    <div className="w-1/2 pr-3">
+                        <h1 className="text-2xl text-white mb-4">Plane Settings</h1>
+
+                        <PlaneSettings />
+                    </div>
+                    <div className="w-1/2 pl-3">
+                        <h1 className="text-2xl text-white mb-4">Audio Settings</h1>
+                        <SoundSettings />
+
+                        <h1 className="text-2xl text-white mt-5 mb-4">flyPad Settings</h1>
+                        <FlyPadSettings />
+
+                        <h1 className="text-4xl text-center text-gray-700 pt-10">flyPadOS</h1>
+                        <h1 className="text-xl text-center text-gray-600 py-2">vAlpha</h1>
+                        <h1 className="text-md text-center text-gray-700 py-2">Copyright &copy; 2020-2021 FlyByWire Simulations</h1>
```

Comment:
how about {(new Date().getFullYear())} :D

Reply:
You should really only add new dates if you substantially change the work.